crates/runner-host/crates/web-client/Cargo.toml

```diff
@@ -21,3 +22,11 @@ yew-hooks = "0.3.2"
 clippy.unit-arg = "allow"
 rust.unsafe-op-in-unsafe-fn = "warn"
 rust.unused-crate-dependencies = "warn"
+
+[features]
+web_client_log_level_info = []
+web_client_log_level_debug = []
+web_client_log_level_warn = []
+web_client_log_level_error = []
+web_client_log_level_trace = []
```

Comment:
You need to put this table before the `[lints]` one, because the sync script assumes `[lints]` is the last one. We can also use much simple feature names:

```toml
[features]
# At most one log level can be selected (defaults to off).
log_debug = []
log_error = []
...
```